.travis.yml

```diff
@@ -85,3 +85,90 @@ deploy:
   on:
     tags: true
     condition: $MINIMAL_ENV = false
+
```

Comment:
Are all these extra empty lines supposed to be here?

Reply:
I don't know how they got in there. Fixed, thanks!